project3/styles.css

```
@@ -0,0 +1,361 @@
+/* Font & Color Palette */
+
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1E2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+
+    --font-body: 'BenchNine', sans-serif;
+    /* Weights: 300, 400, 700 */
+
+    --font-heading: 'Teko', sans-serif;
+    /* Weights: 500, 600, 700 */
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+/* Utility Classes */
+
+.button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.button-reset[disabled] {
+    cursor: default;
+}
+
+/* Global Settings */
+
+body {
+    background-color: var(--color-blue);
+    color: white;
+    font-family: var(--font-body);
+    font-size: 22px;
+    margin: 0;
+}
+
+button[type="submit"] {
+    background-color: var(--color-green);
+    color: black;
+    clear: both;
+    display: block;
+    font-family: var(--font-heading);
+    font-size: 32px;
+    height: 87px;
+    margin: 120px auto 80px auto;
+    text-align: center;
+    text-transform: uppercase;
+    transition: all 300ms;
+    width: 205px;
+}
+
+button:hover {
+    background-color: white;
+    border: 4px solid var(--color-green);
+}
+
+div input {
+    width: 24px;
+}
+
+fieldset {
+    border: 0;
+    padding: 0;
+}
+
+fieldset label {
+    bottom: 8px;
+    position: relative;
+}
+
+fieldset span {
+    display: block;
+}
+
+fieldset+fieldset {
+    margin-top: 34px;
+}
+
+fieldset img {
+    margin-bottom: 18px;
+}
+
+form {
+    background-color: var(--color-offblack);
+    padding: 18px;
+    margin: 520px 24px 24px 24px;
+    max-width: 1000px;
+}
+
+header {
+    background-color: var(--color-grey);
+    height: 600px;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: -1;
+}
+
+h1,
+h2 {
+    font-family: var(--font-heading);
+    text-transform: uppercase;
+}
+
+h1 {
+    color: black;
+    font-size: 48px;
+    font-weight: 700;
+    line-height: 40px;
+    margin: 40px 18px 20px 18px;
```

Can be simplified, if you like:
```suggestion
    margin: 40px 18px 20px;
```